src/components/ActivityFeed/PostHeader.tsx

```
@@ -0,0 +1,380 @@
+
+		const decisionPeriodStartsAt = decidingStatusBlock && decidingStatusBlock.timestamp ? dayjs(decidingStatusBlock.timestamp) : prepare.periodEndsAt;
+		const decision = getPeriodData(network, decisionPeriodStartsAt, trackDetails, 'decisionPeriod');
+		setDecision(decision);
+		setRemainingTime(convertRemainingTime(decision.periodEndsAt));
+
+		// eslint-disable-next-line react-hooks/exhaustive-deps
+	}, []);
+	const totalVotes = ayesNumber + naysNumber;
+	const ayesPercentage = totalVotes > 0 ? (ayesNumber / totalVotes) * 100 : 0;
+	const naysPercentage = totalVotes > 0 ? (naysNumber / totalVotes) * 100 : 0;
+	const isAyeNaN = isNaN(ayesPercentage);
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)